go_covert_lib/controller/processor/asymmetricEncryption/asymmetricEncryption_config.go

```diff
@@ -0,0 +1,25 @@
+package asymmetricEncryption
+
+import (
+	"../../config"
+)
+
+type ConfigClient struct {
+	SenderPublicKey    config.KeyParam
+	SenderPrivateKey   config.KeyParam
+	ReceiverPublicKey  config.KeyParam
+	ReceiverPrivateKey config.KeyParam
+}
+
+func GetDefault() ConfigClient {
+	return ConfigClient{
+		SenderPublicKey:    config.MakeKey([]byte("12345"), config.Display{Description: "Your Public Key"}),

```

Comment:
Please add a name to all Display structs. Refer to https://github.com/RyanAbraham/Covert-Channels/pull/24/files

Reply:
Is group also required?